src/components/App/App.tsx

```diff
@@ -3,12 +3,12 @@ import {QuestionContext, AllQuestionFormat} from '../../types'
 import {getQuestions} from '../../apiCalls'
 
 //components
+import NavBar from '../NavBar/NavBar';
```

Comment:
This file holds functionality for the landing page and also, will import the Survey Questions from BE Server

Reply:
Exactly!, this App React component is the one that fires the API call to populate the survey and holding on to the Survey's current state, from here we will send you the hash you need to get back the report.